fetch.bs

```diff
@@ -5139,12 +5253,14 @@ these steps:
      <a>filtered response</a>; otherwise to <var>response</var>'s
      <a for="filtered response">internal response</a>.
 
+     <li><p>If <var>response</var>'s <a for=response>type</a> is "<code>error</code>" then
+      return the result of running <a>HTTPS upgrade fallback</a> given <var>request</var>
+      and a <a>network error</a>.
```

Comment:
It seems this would return the original response rather than a new network error. That's probably okay though.

Nit: indentation is wrong. We don't further indent for wrapped lines.

Reply:
If response is a network error, the fallback algorithm will create a new 307 response with the fallback URL. Or do you mean `is HTTPS upgrade` isn't set so fallback algorithm returns at step 1?